core/src/point.rs

```diff
@@ -22,6 +22,16 @@ impl Point {
     pub const fn new(x: f32, y: f32) -> Self {
         Self { x, y }
     }
+
+    /// Computes the distance to another [`Point`].
+    ///
+    /// [`Point`]: struct.Point.html
+    pub fn distance(&self, to: Point) -> f32 {
+        let a = self.x - to.x;
+        let b = self.y - to.y;
+
+        f32::sqrt(a * a + b * b)
```

Comment:
Can also use the built-in [`hypot` method](https://doc.rust-lang.org/std/primitive.f32.html#method.hypot)

Return value would be `a.hypot(b)`

Reply:
Oh, nice one. Thank you!